build-farm/platform-specific-configurations/linux.sh

```diff
@@ -137,6 +137,24 @@ if [ "${VARIANT}" == "${BUILD_VARIANT_DRAGONWELL}" ] && [ "$JAVA_FEATURE_VERSION
   fi
 fi
 
+if [ "${VARIANT}" == "${BUILD_VARIANT_LOONGSON}" ]; then
+  if [ "$JAVA_FEATURE_VERSION" -ne 8 ]; then
+    echo "Only Java 8 is supported for now"
+    exit 1
+  fi
+  if [ "${ARCHITECTURE}" == "loongarch64" ]; then
+    echo Loongson jdk8 requires a Loongson boot JDK - downloading one ...
```

Comment:
Why is the architecture check required? Doesn't it require a Loongson JVM to build the Loonson codebase every time?

Reply:
> Why is the architecture check required? Doesn't it require a Loongson JVM to build the Loonson codebase every time?

I found that in jenkins jobs, Loongson codebase will build on x86 machine. If not architecture check, x86 machine will use the LoongArch jdk binary, and an error will occur when VARIANT is BUILD_VARIANT_LOONGSON. Maybe my ideas are wrong and welcome to give your opinion.